src/main/java/net/mcreator/ui/workspace/WorkspacePanel.java

```diff
@@ -1218,52 +1219,92 @@ public void editCurrentlySelectedModElement(ModElement mu, JComponent component,
 	}
 
 	private void editCurrentlySelectedModElementAsCode(ModElement mu, JComponent component, int x, int y) {

```

Comment:
I would suggest moving code for the code selection dropdown to a separate class as this class is slowly becoming a "god" class we don't want

Reply:
✔️ Fixed